File: DAY_44/DAY_44_03.java
// Created by KUNTAL OJHA .
// Today 14/10/2022 (dd/mm/yyyy)
// Problem Link: https://www.codechef.com/submit/CWIREFRAME
// Problem Title: Chef and Wire Frames
// Programmng Language: JAVA



/* package codechef; // don't place package name! */

import java.util.*;
import java.lang.*;
import java.io.*;

/* Name of the class has to be "Main" only if the class is public. */
class Codechef
{
	public static void main (String[] args) throws java.lang.Exception
	{
		// your code goes here
		Scanner kuntal = new Scanner(System.in);
		int t = kuntal.nextInt();
		for(int i=0;i<t;i++)
		{
		    int n = kuntal.nextInt();
		    int m = kuntal.nextInt();
		    int x = kuntal.nextInt();
		    
		    System.out.println(2*(n+m)*x);

		}

	}
}

File: DAY_44/DAY_44_03.py
# Created by KUNTAL OJHA .
# Today 14/10/2022 (dd/mm/yyyy)
# Problem Link: https://www.codechef.com/submit/CARTRIP
# Problem Title: Car Trip
# Programmng Language: PYTHON



def main():
    t = int(input())
    i=0 
    while i<t:
      i=i+1
      
      x =int(input())
      
      if x<=300:
          print("3000")
      else:
          print(x*10)

if __name__=='__main__':
    main()




